coordinator/src/main/scala/com/socrata/datacoordinator/service/collocation/Collocator.scala

```diff
@@ -235,8 +238,17 @@ class CoordinatedCollocator(collocationGroup: Set[String],
                   Rejected(s"the total size in bytes exceeds the limit ${costLimits.totalSizeBytes}")
                 else if (totalCost.getMoveSizeMaxBytes > costLimits.getMoveSizeMaxBytes)
                   Rejected(s"the max move size in bytes exceeds the limit ${costLimits.getMoveSizeMaxBytes}")
-                else
-                  Approved
+                else {
+                  totalMoves.groupBy(_.storeIdTo).flatMap { case (storeId, moves) =>
+                    val bytesToMove = Move.totalCost(moves).totalSizeBytes
+                    val storeTotalBytes = storeMetricsMap(storeId).totalSizeBytes
+                    val storeCapacityBytes = groupConfig.instances(storeId).storeCapacityMB * 1000 * 1000
```

Comment:
This might actually need to be `* 1024 * 1024`, not sure what environment these values are going to be coming from. Might also be worth putting a bit of fudge in there anyway. Instead of filling exactly to capacity only allow collocation to bring it within something like 5% of capacity?

Reply:
I was thinking capacity would already have a margin on it, so no need to add extra logic for that here.